inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp

```diff
@@ -579,6 +579,7 @@ MKLDNNMemoryDesc::operator InferenceEngine::TensorDesc() const {
             order = {0, 1};
             blkDims = dims;
             break;
+        case memory::oiw:
```

Comment:
Why do we need support this dormat?
@dmitry-gorokhov Please have a look.

Reply:
This change allow to pass an ONNX test case which tests ConvTranspose operator with 1d data.